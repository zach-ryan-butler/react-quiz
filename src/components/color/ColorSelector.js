import React from 'react';
import PropTypes from 'prop-types';

function ColorSelector({ addColor }) {
  return (
    <form onSubmit={addColor}>
      <span>Change font color </span>
      <input type="color" />
      <span>Change background color</span>
      <input type="color" />
      <button>Click</button>
    </form>
  );
}

ColorSelector.propTypes = {
  addColor: PropTypes.func.isRequired
};

export default ColorSelector;
