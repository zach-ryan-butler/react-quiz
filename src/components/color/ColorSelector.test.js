import React from 'react';
import { shallow } from 'enzyme';
import ColorSelector from './ColorSelector';

describe('TextSelector component', () => {
  it('renders TextSelector', () => {
    const wrapper = shallow(<ColorSelector
      addColor={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
