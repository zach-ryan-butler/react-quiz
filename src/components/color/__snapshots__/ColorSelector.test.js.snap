// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextSelector component renders TextSelector 1`] = `
<form
  onSubmit={[Function]}
>
  <span>
    Change font color 
  </span>
  <input
    type="color"
  />
  <span>
    Change background color
  </span>
  <input
    type="color"
  />
  <button>
    Click
  </button>
</form>
`;
